### ISSUE TITLE
[iOS] Autofill if login only has password but no username

### ISSUE THREAD

**Issue by xanoni** [bitwarden/android#1494]
## Describe the Bug

On iOS (not 100% sure if I've seen it on Android too) autofill does not work if a login only includes a password but not a username. However, there are certain apps that ask you to e.g. specify an unlock code without a username.

Bitwarden then only offers the user to "copy" the password, but it has to be manually pasted into the application.

The workaround is to just save a dummy username (e.g., "N/A"). This will make Bitwarden properly fill the password field (while ignoring the dummy username).

## Steps To Reproduce

1. Create an account for an app that prompts for only a password
2. Leave the username empty
3. Save and try to autofill the password when you open the app

## Expected Result

Password is autofilled.

## Actual Result

Bitwarden shows a "copy password" button and user has to manually paste.

## Screenshots or Videos

N/A

## Environment

- Device: all iPhoens
- Operating system: all recent iOS
- Build Version (go to "Settings" → "About" in the app): don't currently have an iPhone available to check
- Is this a Beta release? N

## Additional Context

N/A

**Comment by miguelarios** [comment]
Was coming to submit this but. I don’t know if it is by design but it shouldn't. Feels the auto fill should work for username and password or just one of the other.

**Comment by xanoni** [comment]
@miguelarios Not sure if you've seen it, but try the workaround that I mentioned. I guess this is why it hasn't been fixed, yet ... easy to work around it.

> The workaround is to just save a dummy username (e.g., "N/A").

**Comment by miguelarios** [comment]
Hey! Yes I have done that workaround. I discovered on my own before I found the ticket. It is a nice workaround but still wanted to support your ticket as a quality of life problem 😀El feb. 3, 2023, a la(s) 3:40 p.m., Linus Xanoni ***@***.***> escribió:﻿
@miguelarios Not sure if you've seen it, but try the workaround that I mentioned. I guess this is why it hasn't been fixed, yet ... easy to work around it.

The workaround is to just save a dummy username (e.g., "N/A").
